api/plugin_support.h

```diff
@@ -298,6 +355,21 @@ int mambo_reserve_cc_space(mambo_context *ctx, size_t size);
 mambo_branch_type mambo_get_branch_type(mambo_context *ctx);
 
 /* Symbol-related functions */
+
+/**
+ * @brief Uses the untranslated address of the application's instruction to
+ * return the symbol name, the starting address of the symbol name and the
+ * filename that this instruction belongs to.
+ *
+ * @param addr The untranslated address of the application's instruction.
+ * @param sym_name Pointer to the pointer to the address where the symbol name
+ * will be stored.
+ * @param start_addr Pointer to the pointer to the address where the starting
```

Comment:
I'd simplify this one to: Pointer to a pointer where the starting ...

I know we discussed that, but the current writing makes it confusing when I'm reading it.